apps/cancel-request/translations/src/en/validation.json

```diff
@@ -2,9 +2,12 @@
     "cnc-main-applicant-full-name": {
         "required": "Enter the main applicant's full name",
         "maxlength": "Applicant's full name must be 3 and 250 characters",
-        "": "Applicant's full name must be 3 and 250 characters"
+        "minlength": "Applicant's full name must be 3 and 250 characters"
     },
     "cnc-main-applicant-dob": {
+        "required": "Enter the applicant's date of birth",
+        "before": "Main applicant's date of birth must be in the past",
+        "date": "Enter a real date of birth"
     },
     "cnc-main-applicant-nationality": {
         "required": "Please selet a country"
```

Comment:
'Select the main applicant's country of nationality'  so it matches the other nationality fields for now and removes the typo